agent/functional_tests/util/utils_unix.go

```diff
@@ -260,7 +261,14 @@ func (agent *TestAgent) StartAgent() error {
 // * EXECDRIVER_PATH: the path of metrics
 func (agent *TestAgent) getBindMounts() []string {
 	var binds []string
-	cgroupPath := utils.DefaultIfBlank(os.Getenv("CGROUP_PATH"), defaultCgroupPath)
+	var cgroupPath string
+
+	if runtime.GOARCH == "arm64" {
+		cgroupPath = utils.DefaultIfBlank(os.Getenv("CGROUP_PATH"), defaultCgroupPathAgentMount)
+	} else {
+		cgroupPath = utils.DefaultIfBlank(os.Getenv("CGROUP_PATH"), defaultCgroupPath)
```

Comment:
What is the right cgroup path for AL2? Seems here only arm64 gets updated to a different path?

Reply:
> Could you link the error message here and the test result on AL2 so we are able to know this pr fix the test?

2019-03-29T23:25:33Z [WARN] Disabling TaskCPUMemLimit because agent is unabled to setup '/ecs' cgroup: cgroup create: unable to create controller: cgroups: cgroup mountpoint does not exist

Reply:
> What is the right cgroup path for AL2? Seems here only arm64 gets updated to a different path?

/sys/fs/cgroup